src/index.ts

```diff
@@ -61,9 +61,14 @@ function getTransform(formats: (string | number)[], strings: string[], dict?: IM
             for (let i = 0; i < children.length; i++) {
                 const ch = children[i];
                 if (ch.children) {
-                    let value = ch.children.v.value;
-                    if (ch.attribs.t === 's') {
-                        value = strings[value];
+                    let value;
```

Comment:
Change to `let value: any;` so compiler could build this without issues.

Reply:
Done, thank you!